File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/LoginResponsePacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class LoginResponsePacket extends Packet {

    public static final int SUCCESS = 0;
    public static final int DENIED = 1;

    public Object responseCode = 0;

    @Override
    public void encode() {
        super.encode();
        put(responseCode);
    }

    @Override
    public void decode() {
        super.decode();
        responseCode = get();
    }

    @Override
    public Integer getId() {
        return ID_LOGIN_RESPONSE;
    }

    public static LoginResponsePacket create(int responseCode) {
        LoginResponsePacket pk = new LoginResponsePacket();
        pk.responseCode = responseCode;
        return pk;
    }
}

File: CloudBridge-PM/src/CloudBridge/network/protocol/packet/LoginResponsePacket.php
<?php

namespace CloudBridge\network\protocol\packet;

class LoginResponsePacket extends Packet {

    const SUCCESS = 0;
    const DENIED = 1;

    public int $responseCode = 0;

    public function encode(): void {
        parent::encode();
        $this->put($this->responseCode);
    }

    public function decode(): void {
        parent::decode();
        $this->responseCode = $this->get();
    }

    public function getId(): int {
        return self::ID_LOGIN_RESPONSE;
    }

    public static function create(int $responseCode): self {
        $pk = new self;
        $pk->responseCode = $responseCode;
        return $pk;
    }
}